docs/GettingStarted.md

```diff
@@ -65,16 +65,19 @@ const selector = ({ color = 'red' } = {}) => ({
 })
 
 // OR operator
-const selector = props => ({
-  color: props.color || 'red'
-})
+const selector = props => {
+  props = props || {}
```

Comment:
Yeah perhaps. It just simplifies rendering base selectors as you can do `fela.render(selector)` instead of `fela.render(selector, {})`. Same goes for plugins which uses an empty array by default, but I am not sure if we should not remove the plugins parameter from render at all since you can pass them to the renderer directly. I do not see any use case where it would be required to render with some additional plugins which are only required for a single selector.